.readthedocs.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 build:
     os: ubuntu-22.04
     tools:
-        python: '3.8'
+        python: '3.13'
```

Comment:
Let's try this? If latest py version support, why not? Otherwise we need to adapt this when we drop 3.9 this year.

Reply:
Seems RTD build pass. Let me know what you think and if we are going to use 3.13, please update also the PR in kiwipy, and disk-objectstore. Thanks.